src/models/GlossaryModel.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace statikbe\deepl\models;
+
+use craft\base\Model;
+
+class GlossaryModel extends Model
+{
+    public string $name = '';
+
+    public string $source = '';
+
+    public string $target = '';
+
+    public array $entries = [];
+
+    public $uid;
+
+    public function populate(array $data)
+    {
+        $sites = \Craft::$app->getSites();
+
+
+        $this->source = $data['source'];
```

Comment:
Ik zou hier met ?? en defaults werken of juistheid afdwingen via de method parameters en eventueel een custom exception.

Reply:
Voor elk van deze parameters waar je de data array voor accessed

Reply:
isset() rond gezet + valiation rules voor required/string/array toegevoegd hiervoor!